index.html

```diff
@@ -127,20 +122,21 @@ <h3 class="multi-text-color">Gain+Glory</h3>
 </section>
 
 <!-- About me Session -->
-<section class="about-container">
-    <img class="float-right" src="images/Mask_Group_floate_right.png" alt="elipse-floating-right">  
-      <h2 class="about-me">About Me</h2><br><br><br>
-         <p class="about-message">I can help you build a product, feature or website 
-         Look through some of my work and experience! If you like what you see and have a project you need coded, 
-          don't hestiate to contact me.</p>
-         <div><img class="float-left" src="images/Group_float_left.png" alt="floating-elispe-left">
-  <button class="see-project get-resume" type="button">Get My Resume</button>  
-</div> 
+<section> 
+  <div class="about-container"> 
+    <div class="about-me-background-image-1"><div class="about-me"><h2>About me</h2></div>      
+        <div class="about-background-desktop-image-2">
```

Comment:
- [ ] This sections seems to have been affected by the changes introduced to the desktop version. The `About Me` text is on top of the background image. So, you should update it according to the Figma design.
### Design
![about-me-affected](https://user-images.githubusercontent.com/44057863/166108642-914d0436-68d8-47d3-bd82-046939e7f64e.png)

### Implementation
![about-me-affected-impl](https://user-images.githubusercontent.com/44057863/166108633-459a5d35-69a1-4655-89e9-044a8fd760c1.png)


- [ Optional ] You should background image to the about-me section instead of having a `<div>` for sake of adding background images.  You could have exported the svg image from the Figma design from the componet with the name  of `illustration-about me-mobile` under `Template 3- main-mobile`.

Reply:
Its okay